timbermill-java/timbermill-local/src/main/java/com/datorama/oss/timbermill/ElasticsearchClient.java

```diff
@@ -458,22 +461,22 @@ private void updateOldAlias(RolloverResponse rolloverResponse, String timbermill
 				LOG.error("Removing old index from alias [{}] failed", oldAlias);
 			}
 
-//			Optional<String> oldIndexOptional = oldAliases.keySet().stream().findAny();
-//			if (oldIndexOptional.isPresent()){
-//				String oldIndex = oldIndexOptional.get();
-//				ForceMergeRequest forceMergeRequest = new ForceMergeRequest(oldIndex).maxNumSegments(1);
-//				client.indices().forcemergeAsync(forceMergeRequest, RequestOptions.DEFAULT, new ActionListener<ForceMergeResponse>() {
-//					@Override
-//					public void onResponse(ForceMergeResponse forceMergeResponse) {
-//					}
-//
-//					@Override
-//					public void onFailure(Exception e) {
-//					}
-//				});
-//
-//				LOG.info("Force merge index {} to 1 segment", oldIndex);
-//			}
+			Optional<String> oldIndexOptional = oldAliases.keySet().stream().findAny();
+			if (oldIndexOptional.isPresent()){
+				String oldIndex = oldIndexOptional.get();
+				ForceMergeRequest forceMergeRequest = new ForceMergeRequest(oldIndex).maxNumSegments(1);
+				client.indices().forcemergeAsync(forceMergeRequest, RequestOptions.DEFAULT, new ActionListener<ForceMergeResponse>() {
```

Comment:
add logs for start / success / failure, would be nice to also get a time measurement maybe even report it to kamon so we can correlate this action to other issues if such arise.